src/config/config.js

```diff
@@ -106,6 +106,7 @@ const Config = {
   torusGoogleAuth0: env.REACT_APP_TORUS_GOOGLEAUTH0 || 'google-auth0-gooddollar',
   torusAuth0SMS: env.REACT_APP_TORUS_AUTH0SMS || 'gooddollar-auth0-sms-passwordless',
   torusEmailEnabled: env.REACT_APP_TORUS_AUTH0EMAIL_ENABLED === 'true',
+  jwtMaxAge: env.REACT_APP_JWT_MAX_AGE || 1000 * 60 * 60 * 24, // 24hours
```

Comment:
@sirpy @serdiukov-o-nordwhale 
what is the expiration time of the JWT?

Reply:
@omerzam 
basically token expiration should be set on the server  side
read https://www.npmjs.com/package/jsonwebtoken 
on server add to .sign the expiration (login-middleware), verify it handles expiration correctly (ExtractJWT from passport-jwt)
on client i think you simply read the expiration from the token, you dont need to add jwtMaxAge

Reply:
> @omerzam
> basically token expiration should be set on the server side

Agreed